### DISCUSSION TITLE
Different results from different sources

### DISCUSSION THREAD

**Discussion by nickpending** [projectdiscovery/asnmap#326] [Ideas]
Trying to figure out why the data returned from asnmap is different from the data provided by bgpview.

With asnmap I receive 7 prefixes:

```
> asnmap -a AS6431 -silent
12.106.32.0/22
204.178.0.0/20
207.76.218.0/24
207.140.168.0/24
192.20.225.0/24
135.197.0.0/16
135.205.0.0/16
135.207.0.0/16
```

and with bgpview I receive 17 prefixes:

`https://bgpview.io/asn/6431#prefixes-v4`

Any idea what's happening in the background here?

**Comment by ehsandeep** [comment]
@nickpending asnmap data is based on and limited to what available at https://iptoasn.com/

**Comment by nickpending** [comment]
Okay, thanks. I did some analysis and it seems that some data in iptoasn.com is summarized. While summarized data can be useful in certain use cases, I think detailed data would be preferred? I wonder if that's something that should be considered for future releases?
